### DISCUSSION TITLE
Quarkus CLI - Deploy

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#45088] [Q&A]
It'll be great to have a deploy method in the quarkus cli - something on the lines of [kamal](https://kamal-deploy.org/) , that deploys on a bare metal ..

[a comment by an automated account is not shown]

**Comment by maxandersen** [comment]
are you suggestion quarkus supports deploy to/via kamal or that quarkus makes an alternative to kamal?

the latter is non-trivial but the former I could see someone definitely contribute - it most likely already just works :)

**Reply by akil-rails** [reply]
the former, I was suggesting a deployment option to VM's , like there are extensions to deploy to openshift and kubernetes..

**Reply by maxandersen** [reply]
Yeah so soemone who knows Kamal should be able to make a quarkus-deploy-kamal plugin to the cli. 

Want to give it a go?

**Reply by akil-rails** [reply]
> Yeah so soemone who knows Kamal should be able to make a quarkus-deploy-kamal plugin to the cli. 
> 
> Want to give it a go?

Yes , will give it a try . I am able to run Kamal to deploy an image built by Quarkus on digital ocean , was pretty smooth - one question though, Kamal is in ruby so the extension would probably just execute Kamal commands isn't it ?
